<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerConfigDlg</class>
 <widget class="QDialog" name="ServerConfigDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Server configuration</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="6">
    <widget class="QPushButton" name="pbAccept">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="7">
    <widget class="QLineEdit" name="leUserName">
     <property name="placeholderText">
      <string>Set the user name</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="lTestResult">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="pbCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="7">
    <widget class="QComboBox" name="cbServer"/>
   </item>
   <item row="6" column="4">
    <widget class="QPushButton" name="pbTest">
     <property name="text">
      <string>Test token</string>
     </property>
    </widget>
   </item>
   <item row="6" column="5">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="7">
    <widget class="QLineEdit" name="leEndPoint">
     <property name="placeholderText">
      <string>Set the repository endpoint</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="7">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="leUserToken">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Set the user token</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ButtonLink" name="lAccessToken">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ButtonLink</class>
   <extends>QLabel</extends>
   <header>ButtonLink.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leUserName</tabstop>
  <tabstop>leUserToken</tabstop>
  <tabstop>cbServer</tabstop>
  <tabstop>leEndPoint</tabstop>
  <tabstop>pbTest</tabstop>
  <tabstop>pbAccept</tabstop>
  <tabstop>pbCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
